jac-uk/apply
# #1120 BUG: Log in page. Password show/hide button is being called instead of form submit

## What's included?
Added button types to ensure hitting the Enter button triggers the submit instead of the show/hide pwd.

Closes #1120 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Got to the login page.
- FIll in the user name.
- Fill in the password and press the Enter button on your keyboard.
- The form should submit instead of the Show/Hide button being triggered.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
